app/src/main/java/io/zenandroid/onlinego/reusable/ActiveGameItem.kt

```diff
@@ -16,6 +16,7 @@ import kotlinx.android.synthetic.main.item_active_game_card.*
 
 class ActiveGameItem (val game: Game) : Item(game.id) {
     override fun bind(holder: ViewHolder, position: Int) {
+        holder.board.isGameCard = true
```

Comment:
I would rather not have this, as the board game doesn't need to know the context it is used (e.g. logic such as "if you are in a game card, then never show coordinates" should not reside inside the board, but inside the game card holder). If tomorrow we want to change it so that game cards DO show coordinates, we want to be able to do that without changing the board view.